library/src/main/scala/giter8/GitRepository.scala

```diff
@@ -47,7 +47,7 @@ object GitRepository {
     val NativeUrl: Regex = "^(git[@|://].*)$".r
     val HttpsUrl: Regex  = "^(https://.*)$".r
     val HttpUrl: Regex   = "^(http://.*)$".r
-    val SshUrl: Regex    = "^(ssh://.*)$".r
+    val SshUrl: Regex    = "^ssh://(.*)$".r
```

Comment:
I am kind of confused by this change.
I think the original issue was that ssh wasn't in the regex at all - https://github.com/foundweekends/giter8/blob/v0.6.8/app/src/main/scala/apply.scala#L23-L32
Doesn't this change drop ssh?